metadata-ingestion/src/datahub/ingestion/source/datahub/datahub_database_reader.py

```diff
@@ -38,14 +38,15 @@ def __init__(
     def query(self) -> str:
         # May repeat rows for the same date
         # Offset is generally 0, unless we repeat the same createdon twice
+
+        # Ensures stable order, chronological per (urn, aspect)
+        # Version 0 last, only when createdon is the same. Otherwise relies on createdon order
```

Comment:
Postgres comments use a different syntax lol

Reply:
just for quoting right?

Reply:
Postgres comments use `--` or `/* */` while mysql supports those plus `#`. I moved the comments completely out of the query string so that we don't have to worry about it